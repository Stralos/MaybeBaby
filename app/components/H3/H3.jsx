/* @flow */
import styled from 'styled-components';

const H3 = styled.h3`
  display: flex;
  justify-content: center;
  font-size: 1.75rem;
  margin-bottom: 24px;
  font-family: 'Marcellus SC';
`;

export default H3;
