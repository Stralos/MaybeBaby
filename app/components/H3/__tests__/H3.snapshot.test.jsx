/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import H3 from '../H3';

describe('H3 Snapshot Testing --', () => {
  it('should render and match snapshot', () => {
    const component = renderer
      .create(<H3 />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
