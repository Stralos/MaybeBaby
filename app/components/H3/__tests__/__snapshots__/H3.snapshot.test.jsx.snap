// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H3 Snapshot Testing -- should render and match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 1.75rem;
  margin-bottom: 24px;
  font-family: 'Marcellus SC';
}

<h3
  className="c0"
/>
`;
